src/components/common/Stepper.vue

```diff
@@ -22,7 +22,8 @@
           >
             <v-icon class="step__icon" :class="{ 'selected-icon': isCurrentStep(step) }">{{ step.icon }}</v-icon>
           </v-btn>
-          <v-icon class="step__btn2" size="30" color="success darken-1" v-show="isValid(step.to)">
+          <v-icon class="step__btn2" size="30" color="success darken-1" v-show="isValid(step.to)
+            && isFirmReviewValid(step.to)">
```

Comment:
Added "isFirmReviewValid(step.to)" to make sure "The Review and Confirm icon should show neutral icon" (from ticket [13372](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/13372)).

Reply:
I'm a little wary of this very firm explicit code in whats is a fairly common component. 

Maybe i'm off but shouldn't `step.to` (ie `isRegistrationValid` in this case) be a computed value that already encapsulates these conditions. 

If you only want to show the invalid icons on the stepper once a user has tried to submit, isn't that just a matter of adding the `validateSteps` flag or whichever flag is triggered, to the condition outlined below in the `isValid()` method?

Reply:
The problem is that `isRegistrationValid` validates all five components. The cross icon will be displayed when the user visits the Review and Confirm step for the first time, because the Certify or StaffPayment component is not checked yet.

Reply:
I agree with Cam. We want to stay away from special cases, and I'm not aware that firms are a special case in this project.

Also, this app's architecture (global functionality) should apply to all entity types. If this bug applies to all entity types then this code doesn't fix it. If this bug applies to firms only then there's something broken in the firms code that should be fixed instead.